src/relations.jl

```diff
@@ -2915,9 +2915,10 @@ function virtual_dry_static_energy(
     ts::ThermodynamicState{FT},
     e_pot::FT,
 ) where {FT <: Real}
+    T_0::FT = TP.T_0(param_set)
     T_virt = virtual_temperature(param_set, ts)
     _cp_m = cp_m(param_set, ts)
-    return _cp_m * T_virt + e_pot
```

Comment:
@tapios I know this is not important but since I am changing it anyway, should I change the cp_m to cp_d here?

Reply:
Yes, please.